### ISSUE TITLE
Coreaudio compilation failure - bindings generation fail due to old rustfmt bug

### ISSUE THREAD

**Issue by saresend** [RustAudio/coreaudio-sys#7]
Hey, I'm using coreaudio-sys as a dependency for [CPAL](https://github.com/tomaka/cpal), and it seems that the pinned v 0.2.0 is failing to compile with the following error: 

```
error: unexpected close delimiter: `]`
    --> /Users/.../coreaudio-sys-a6bbc6ed7b769743/out/coreaudio.rs:2040:33
     |
2040 | #[derive(Copy, Clone)]; 16usize ] , }#[test]
     |                                 ^

error: Could not compile `coreaudio-sys`.
```

I can't find other sources of people having this issue, so I'm not sure if it is on my end or not. 
Thanks!

**Comment by tomaka** [comment]
Indeed I don't think it's a good idea to run rustfmt I'm generated bindings which nobody is going to read anyway!

**Comment by saresend** [comment]
Does cargo automatically run rustfmt on dependencies?

**Comment by alexander-irbis** [comment]
A simple workaround if someone can not update the rustfmt
```
--- a/build.rs
+++ b/build.rs
@@ -73,6 +73,7 @@ fn main() {
     let bindings = builder
         .trust_clang_mangling(false)
         .derive_default(true)
+        .rustfmt_bindings(false)
         .generate()
         .expect("unable to generate bindings");
```

**Comment by Rhuagh** [comment]
Bindgen by default automatically run rust fmt on the generated bindings. This pr disables that automatic fmt

**Comment by Rhuagh** [comment]
Fixed by #13 .
